docs/features/software-templates/authorizing-scaffolder-tasks-parameters-steps-and-actions.md

```diff
@@ -174,7 +174,64 @@ class ExamplePermissionPolicy implements PermissionPolicy {
 }
 ```
 
-Although the rules exported by the scaffolder are simple, combining them can help you achieve more complex cases.
+### Authorizing scaffolder tasks
+
+The scaffolder plugin also exposes permissions that can restrict access to tasks, task logs, task creation, and task cancellation. This can be useful if you want to control who has access to the scaffolder.
```

Comment:
```suggestion
The scaffolder plugin also exposes permissions that can restrict access to tasks, task logs, task creation, and task cancellation. This can be useful if you want to control who has access to these areas of the scaffolder.
```

Reply:
Maybe I'm wrong and these permissions can do more then I'm understanding from the docs you wrote and my quick look at the code. In the example you would still see the template but not right them, right?

Reply:
I think that doc change makes sense. So the read permission just restrict viewing the resultant tasks that are generated when you trigger a scaffolder template. The create permission prevents the scaffolder templates from being executed, which in turns prevents creation of tasks. Cancel permission just prevents cancellation of running tasks. Seeing the templates themselves are controlled by the catalog read permissions.